go.mod

```diff
@@ -1,27 +1,29 @@
 module sigs.k8s.io/custom-metrics-apiserver
 
-go 1.20
+go 1.21
+
+toolchain go1.21.5
 
 require (
 	github.com/emicklei/go-restful/v3 v3.11.0
 	github.com/google/addlicense v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
-	k8s.io/api v0.28.1
-	k8s.io/apimachinery v0.28.1
-	k8s.io/apiserver v0.28.1
-	k8s.io/client-go v0.28.1
-	k8s.io/component-base v0.28.1
-	k8s.io/klog/v2 v2.100.1
-	k8s.io/kube-openapi v0.0.0-20230901164831-6c774f458599
-	k8s.io/metrics v0.28.1
-	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
+	k8s.io/api v0.28.5
```

Comment:
Since v0.29.0 is out now, do you think you could update to the new minor instead of the latest patch?

Reply:
I would like to have a version with 0.28 deps so we can use it in KEDA's new release. I am happy to open another PR with 0.29 if https://github.com/kubernetes-sigs/custom-metrics-apiserver/pull/162 doesn't go well.

Reply:
sounds good :+1:

Reply:
@zroubalik Given #162 has been closed, I wonder would you like to open another PR with v0.29?